apps/studio/src/components/Searchbar/SearchResult.tsx

```diff
@@ -0,0 +1,151 @@
+import { type ReactNode } from "react"
+import { Box, HStack, Icon, Skeleton, Text, VStack } from "@chakra-ui/react"
+import { ResourceType } from "~prisma/generated/generatedEnums"
+
+import type { SearchResultResource } from "~/server/modules/resource/resource.types"
+import { ICON_MAPPINGS } from "~/features/dashboard/components/DirectorySidebar/constants"
+import { getLinkToResource } from "~/utils/resource"
+
+const formatDate = (date: Date): string => {
```

Comment:
actually i can't seem to find this elsewhere in the codebase

Reply:
Should be this one: https://github.com/isomerpages/isomercms-frontend/blob/develop/src/utils/dateUtils.ts

I think mainly is that I think this is a common function that may be used elsewhere so best is to have it as a util function rather than keep it inside this file.

Reply:
addressed in https://github.com/opengovsg/isomer/pull/882/commits/03cf9643123fe37709fb51620197963eb99e9005